### PR TITLE
EPMRPP-90680 || Popover is broken in sidebar

### PR DESCRIPTION
**Pull request by BlazarQSO**
## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/layouts/common/appSidebar/appSidebar.scss [hunk 1]
```diff
@@ -121,6 +121,14 @@
...
+
+    button div {
+      color: $COLOR--white-two;
+
+      path {
+        fill: $COLOR--white-two;
+      }
+    }
   }
 }
```
Comment (AmsterGet) [inline app/src/layouts/common/appSidebar/appSidebar.scss:133]: Looks as a duplicate of lines 135-147.
Can in be handled via single set of such styles?

Another comment is about using relative tags selectors (button > div).
Can we avoid them here in favor of classes to less depend on child components HTML structure?

Comment (BlazarQSO) [reply app/src/layouts/common/appSidebar/appSidebar.scss]: 1 Done.
2 These changes will be in the component from the component library, so it's better to leave it that way.

File: app/src/layouts/common/appSidebar/userControl/userControl.scss [hunk 2]
```diff
@@ -103,9 +92,11 @@
 
 .popover-control {
   height: 48px;
-  width: 100%;
+  width: 264px;
```
Comment (Vadim73i) [inline app/src/layouts/common/appSidebar/userControl/userControl.scss:95]: Is it ok to use fixed width?

Comment (BlazarQSO) [reply app/src/layouts/common/appSidebar/userControl/userControl.scss:95]: A fixed width is used

**Review by Vadim73i — APPROVED** [review]

**Pull request merged**